single-person.php

```diff
@@ -113,23 +113,25 @@
 							$id = $mypod -> field('id');
 							$permalink = get_permalink($id);
 							$personnel = $mypod -> field('project_personnel.ID');
+							$Live = "Live";
```

Comment:
rather than calling them simply things like`$Live`, please prefix them to something that distinguishes them as something unique to our project. such as `$ODHProjectStatus_Live`. WordPress has never  done proper namespacing (although PHP added it since then), so let's follow the WordPress idiom of prefixing our variables. This is particularly important since these will be used in multiple places.